File: Beginner/Primality Test/priTest.js
const readline = require('prompt-sync')({sigint: true});
const n = Number(readline());
for (let i=0; i<n; i++) {
  let isprime=0;
  let num = Number(readline());
  for(let j=1; j<num+1; j++) {
    if (num % j === 0) {
      isprime += 1;
    }
  }
  if (isprime === 2) {
    console.log("yes\n");
  } else {
    console.log("no\n");
  }
} 

File: Beginner/Primality Test/priTest.py
# cook your dish here
for i in range(int(input())):
    number = int(input())
    is_prime = 0
    for j in range(1, number+1):
        if number % j == 0:
            is_prime += 1
    if is_prime == 2:
        print("yes")
    else:
        print("no")
